iac-extensions/docker/src/test/resources/checks/MalformedJsonInExecCheck/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM scratch
+
+# Checking any instructions that can contains shell forms
+# Noncompliant@+1
+CMD [ not a valid json
+# Noncompliant@+1
+RUN [ not a valid json
+# Noncompliant@+1
+ENTRYPOINT [ not a valid json
+# Noncompliant@+1
+ADD [ not a valid json
+# Noncompliant@+1
+COPY [ not a valid json
```

Comment:
What if command contains single `]`?

Reply:
I added the test case to document it.
It will not raise an issue because we have the main predicate which is `command.startsWith("[")`